cypress/e2e/signIn.cy.js

```diff
@@ -1,7 +1,20 @@
 /// <reference types="cypress" />
 
 describe('Sign In page', () => {
+  const username = 'delta2024';
+  const email = 'delta2024@ukr.net';
+  const password = 'cvRejected2024';
```

Comment:
Consider storing test data like username, email, and password in a separate configuration file or use environment variables. This approach improves maintainability and keeps sensitive information out of the test scripts.